### PR TITLE
Feat/new dashboard edit tags

### PR DESCRIPTION
**Pull request by mpisacic**

### CODE REVIEW

File: src/components/CompactDashboard/CompactDashboard.Props.ts [hunk 1]
```diff
@@ -35,7 +36,7 @@ export interface ICompactDashboardProps {
   /**
   * Action that is called on role change of a certain server of some farm. The function is supplied with server FQDN.
   */
-  serverRoleEdit?: (serverFQDN: any) => void;
+  serverRoleEdit?: (event: any, serverFQDN: any, farmId: any) => void;
```
Comment (hbagarich) [inline src/components/CompactDashboard/CompactDashboard.Props.ts]: You can rename serverFQDN to serverId maybe?

File: src/components/CompactFarm/CompactFarm.Props.ts [hunk 2]
```diff
@@ -8,5 +8,8 @@ export interface ICompactFarmProps {
   iconName?: string;
   serverOnClick?: (groupId: any, serverId: any) => void;
   groupOnClick?: (groupId: any) => void;
+  editRoles?: boolean;
```
Comment (hbagarich) [inline src/components/CompactFarm/CompactFarm.Props.ts]: Maybe rename this to allowEditRoles/showEditRoles?

File: src/components/CompactServer/CompactServer.Props.ts [hunk 3]
```diff
@@ -16,4 +16,6 @@ export interface ICompactServerProps {
     onClose?: (serverId: string) => void;
     onMouseEnter?: React.EventHandler<React.MouseEvent<HTMLDivElement>>;
     onMouseLeave?: React.EventHandler<React.MouseEvent<HTMLDivElement>>;
+    editRoles?: boolean;
```
Comment (hbagarich) [inline src/components/CompactServer/CompactServer.Props.ts]: Same rename suggestion here as well, editRoles-> allowEditRoles/showEditRoles

File: src/components/CompactFarm/CompactFarm.Props.ts (hunk 2 above)
Comment (vinkobedek) [inline src/components/CompactFarm/CompactFarm.Props.ts]: i second this suggestion. better to see that it is a boolean directly from the name

**Pull request merged**